src/components/vr_cooperation/include/vr_cooperation/commands/base_json_request.h

```diff
@@ -128,6 +128,12 @@ class BaseJsonRequest : public BaseCommandRequest,
   virtual void SendMessageToMobile(application_manager::MessagePtr message);
 
   /**
+   * @brief send timeout response to hmi
+   * @param message gpb message to hmi
+   */
+  virtual void SendOnTimeoutResponse(vr_hmi_api::ServiceMessage& message);
```

Comment:
no, some common part (rpc, rpc_type, correlation_id) of message will be set inside function